libp2p/protocols/secure/secure.nim

```diff
@@ -80,9 +80,9 @@ method init*(s: Secure) {.gcsafe.} =
 
   s.handler = handle
 
-method secure*(s: Secure, conn: Connection): Future[Connection] {.async, base, gcsafe.} =
+method secure*(s: Secure, conn: Connection, initiator: bool): Future[Connection] {.async, base, gcsafe.} =
   try:
-    result = await s.handleConn(conn, true)
+    result = await s.handleConn(conn, initiator)
```

Comment:
`secure` is explicitly an initiator, we should leave it like that.

Reply:
That was the whole issue with this PR, sadly it's not, check `switch.nim` implementation